cpp/ql/lib/semmle/code/cpp/AutogeneratedFile.qll

```diff
@@ -106,6 +106,8 @@ private int fileHeaderLimit(File f) {
         )
         or
         line = f.getMetrics().getNumberOfLines()
+        or
+        line = 1000
```

Comment:
I think it would be good to have a small comment about why we need this additional disjunct when we already have `line = f.getMetrics().getNumberOfLines()` disjunct.

Reply:
Done.